api-tests/src/test/resources/org/openmrs/module/reporting/include/ReportTestDataset-openmrs-1.10.xml

```diff
@@ -523,11 +523,21 @@
                            properties="#Mon Jan 21 14:08:47 CET 2013" creator="1"
                            date_created="2013-01-21 14:08:47" retired="0" />
 
-  <reporting_report_request id="1" uuid="h8a82b63-1066-4c1d-9b43-b405851fc467" report_definition_uuid="c11f5354-9567-4cc5-b3ef-163e28873926"
-                            process_automatically="1" priority="LOW" status="REQUESTED" requested_by="1" request_datetime="2013-01-21 14:08:47" />
-  <reporting_report_request id="2" uuid="d9a82b63-1066-4c1d-9b43-b405851fc467" report_definition_uuid="d11f5354-9567-4cc5-b3ef-163e28873926"
-                            process_automatically="1" priority="LOW" status="REQUESTED" requested_by="1" request_datetime="2013-01-21 14:08:47" />
-  <reporting_report_request id="3" uuid="b0a82b63-1066-4c1d-9b43-b405851fc467" report_definition_uuid="c11f5354-9567-4cc5-b3ef-163e28873926"
-                            process_automatically="1" priority="LOW" status="REQUESTED" requested_by="1" request_datetime="2013-01-21 14:08:47" />
-
-</dataset>
\ No newline at end of file
+  <reporting_report_request id="1" uuid="h8a82b63-1066-4c1d-9b43-b405851fc467"
```

Comment:
Did you intentionally change the formatting of these xml dataset files? I notice that you reformatted quite a number of them.

Reply:
I've added 4th _reporting_report_request_ to all datasets (one for each _openMRSMinorVersion_) and intentionally formatted the _reporting_report_request_ elements group to have the same style as _reporting_report_design_  elements group just a line above - IMO made sense to do.